### PR TITLE
TUP-231: Part 1 of N to Core-UI (Fix Core-CMS / Core-Styles Mismatch)

### PR DESCRIPTION
**Pull request by wesleyboar**
### Overview / Changes

Move back CMS-specific styles from [Core-Styles] repo to [Core-CMS] repo.

<details><summary>Background</summary>

I moved _all_ styles from Core-CMS to Core-Styles. That included some CMS-specific styles. Now I bring them back to CMS.

</details>

Also, better documented current[^1] state of `<tacc-search-bar>`.

[^1]: State changes (improves) via https://github.com/TACC/Core-CMS/issues/101.

### Related

- [TUP-231](https://jira.tacc.utexas.edu/browse/TUP-231)
- required by https://github.com/TACC/Core-CMS/pull/482

### Screenshots

N/A

### Testing

See https://github.com/TACC/Core-CMS/pull/482.

[Core-CMS]: https://github.com/TACC/Core-CMS
[Core-Styles]: https://github.com/TACC/Core-Styles

### CODE REVIEW

File: source/_imports/elements/tacc-search-bar.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# TACC - Stylesheets - Search Bar
+
+The actual search bar markup (in TACC/Core-CMS) currently has a caveat (that will be resolved via [TACC/Core-CMS#101]).
```
Comment (wesleyboar) [inline source/_imports/elements/tacc-search-bar.md:3]:
```suggestion
The search bar (as implemented in TACC/Core-CMS) _currently_ has a caveat (that will be resolved via [TACC/Core-CMS#101]).
```

File: source/_imports/elements/tacc-search-bar.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Certain styles need to load **after** all search bar styles and markup:
+
+```css
+:host { visibility: visible; height: auto; }
+```
+
+This avoids [FOUC] by hiding the search bar _until_ all styles and markup are loaded.
+
+
+
```
Comment (wesleyboar) [inline source/_imports/elements/tacc-search-bar.md:20]:
```suggestion

```

Comment (wesleyboar) [reply source/_imports/elements/tacc-search-bar.md:20]: chore(tup-231): delete excess new lines

**Review by wesleyboar — COMMENTED** [review]
notes for reviewers

**Review by taoteg — APPROVED** [review]
LGTM.

**Comment by wesleyboar** [comment]
Merged in `main`. Went smooth. Merging because this and Core-CMS PR are approved.

**Pull request merged**